-1

Tested at: 4eb4c2c96a863681e4b20aad04663e90969be65e

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-16748/16577/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/Utilities/StaticAnalyzers/src/CmsException.cpp 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/Utilities/StaticAnalyzers/src/edmChecker.cpp 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/Utilities/StaticAnalyzers/src/dablooms.c 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/Utilities/StaticAnalyzers/src/murmur.c 
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/RecoTracker/TkHitPairs/src/InnerDeltaPhi.cc: In member function 'PixelRecoRange<float> InnerDeltaPhi::phiRange(const Point2D&, float, float) const':
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/RecoTracker/TkHitPairs/src/InnerDeltaPhi.cc:233:3: error: 'float32x4_t' was not declared in this scope
   float32x4_t num{dHitmag,dLayer,theROrigin * (dHitmag-dLayer),1.f};
   ^
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2016-11-23-2300/src/RecoTracker/TkHitPairs/src/InnerDeltaPhi.cc:234:15: error: expected ';' before 'den'
   float32x4_t den{2*theRCurvature,2*theRCurvature,dHitmag*dLayer,1.f};
               ^
</pre>

--- a/RecoTracker/TkHitPairs/src/InnerDeltaPhi.cc
+++ b/RecoTracker/TkHitPairs/src/InnerDeltaPhi.cc
@@ -230,8 +230,8 @@ PixelRecoRange<float> InnerDeltaPhi::phiRange(const Point2D& hitXY,float hitZ,fl
   }
 
 #ifdef USE_VECTORS_HERE 
-  float32x4_t num{dHitmag,dLayer,theROrigin * (dHitmag-dLayer),1.f};
-  float32x4_t den{2*theRCurvature,2*theRCurvature,dHitmag*dLayer,1.f};
+  cms_float32x4_t num{dHitmag,dLayer,theROrigin * (dHitmag-dLayer),1.f};
+  cms_float32x4_t den{2*theRCurvature,2*theRCurvature,dHitmag*dLayer,1.f};
   auto phis = f_asin07f(num/den);
   phis = phis*dLayer/(rLayer*cosCross);
   auto deltaPhi = std::abs(phis[0]-phis[1]);

